### DISCUSSION TITLE
How to debug run test script?

### DISCUSSION THREAD

**Discussion by code-brewer** [PaulRBerg/hardhat-template#173] [Q&A]
Hi all.
 I am trying to debug run test script(written in typescript as this template used) by command:
 `  npx --node-options=--inspect  hardhat test  --network rinkeby `
 but it did not stop at startup and `debugger` statement.
 Can your guys guide me how to do this?

Thanks

**Comment by PaulRBerg** [comment]
Hey @code-brewer, the Rinkeby network is deprecated. You should use Goerli or Sepolia instead.

Also, you might want to post your question in the [Hardhat Discord server](https://hardhat.org/discord).

**Reply by code-brewer** [reply]
yeah. what I actually used is `sepolia`. The network isn't the focus, I have tried other network also, all failed run into debug mode. 
Thanks for your info about discord server, I will try.
